python/servo/build_commands.py

```diff
@@ -412,4 +419,4 @@ def clean(self, manifest_path=None, params=[], verbose=False):
             opts += ["-v"]
         opts += params
         return check_call(["cargo", "clean"] + opts,
-                          env=self.build_env(), cwd=self.servo_crate(), verbose=verbose)
+                          env=self.build_env(), cwd=self.servo_crate(android), verbose=verbose)
```

Comment:
I don’t know if this is needed for `clean`, since `cwd` mostly affects optional feature selection.